components/eam/cime_config/config_component.xml

```diff
@@ -106,24 +107,29 @@
       <value compset="RCP4_EAM"                         >2006-2100_cam5_rcp45</value>
       <value compset="RCP6_EAM"                         >2006-2100_cam5_rcp60</value>
       <value compset="RCP8_EAM"                         >2006-2100_cam5_rcp85</value>
-      <value compset="1850(?:SOI)?_EAM.*CMIP6"          >1850_eam_CMIP6</value>
-      <value compset="1850(?:SOI)?_EAM.*CMIP6-1pctCO2"  >1850_eam_CMIP6-1pctCO2</value>
-      <value compset="1950(?:SOI)?_EAM.*CMIP6"          >1950_eam_CMIP6</value>
-      <value compset="1850(?:SOI)?_EAM.*CMIP6.*_BGC%*"  >1850_eam_CMIP6_bgc</value>
-      <value compset="2010(?:SOI)?_EAM.*CMIP6"          >2010_eam_CMIP6</value>
+      <value compset="1850(?:SOI)?_EAM.*CMIP6"          >1850_eam_CMIP6_chemUCI-Linoz-mam5-vbs</value>
```

Comment:
I thought we had decided to have "1850_eam_CMIP6" as the always default config?  Why not just edit that?